docs/examples/affinity/cookie/README.md

```diff
@@ -57,14 +59,16 @@ Last-Modified: Tue, 24 Jan 2017 14:02:19 GMT
 ETag: "58875e6b-264"
 Accept-Ranges: bytes
 ```
-In the example above, you can see a line containing the 'Set-Cookie: INGRESSCOOKIE' setting the right defined stickiness cookie.
-This cookie is created by NGINX, it contains the hash of the used upstream in that request and has an expires. 
-If the user changes this cookie, NGINX creates a new one and redirect the user to another upstream.
 
-If the backend pool grows up NGINX will keep sending the requests through the same server of the first request, even if it's overloaded.
+In the example above, you can see that the response contains a `Set-Cookie` header with the settings we have defined.
+This cookie is created by NGINX, it contains the hash of the upstream used for that request and has an `Expires`.
+If the user changes this cookie, NGINX creates a new one and redirects the user to another upstream.
 
-When the backend server is removed, the requests are then re-routed to another upstream server and NGINX creates a new cookie, as the previous hash became invalid.
+If the backend pool grows NGINX will keep sending the requests through the same server of the first request, even if it's overloaded.
+
+When the backend server is removed, the requests are re-routed to another upstream server and NGINX creates a new cookie, as the previous hash became invalid.
```

Comment:
> creates a new cookie, as the previous hash became invalid.

This is also from older implementation. Now we don't create a new cookie, it'll just pick another upstream server and start sending all subsequent requests to that one (because of consistent hashing).